### DISCUSSION TITLE
nnn always open my text file with LibreOffice.

### DISCUSSION THREAD

**Discussion by laowalter** [jarun/nnn#1720] [General]
I have $EDITOR set, and with the official default config, all the text file opened by LibreOffice.  

```
 echo $EDITOR
nvim
```

Thanks for any reply.

**Comment by N-R-K** [comment]
> all the text file opened by LibreOffice.

If you don't have a custom opener then nnn will just invoke `xdg-open`. Any of the following should solve your issue 

* Customize xdg defaults to use nvim for opening text
* Or use a custom opener
* Or use the `-e` flag

**Comment by laowalter** [comment]
Thanks.
